.cargo/config

```diff
@@ -0,0 +1,3 @@
+[build]
+#rustflags = ["-C", "linker=clang"]
+rustflags = ["-C", "link-arg=-fuse-ld=lld"]
```

Comment:
I don't think we should do this in the project root but hint at this in the README (using `~/.cargo/config` and same contents) as the lld doesn't ship with rustc at least yet.

Reply:
I had suggested @dvc94ch do this, my bad. I'll remove for now and add it to the README.